docs/jwst.rst

```diff
@@ -54,14 +54,31 @@ All classes share some common attributes:
     available modes well enough to be aware if they are trying to
     simulate an inconsistent or physically impossible configuration. 
 
+Optical Telescope Element (OTE)
+===============================
+
+The JWST Optical Telescope Element consists of the telescope optics that serve all the science instruments and the fine guidance sensor. Most notably, this means the primary, secondary, and tertiary mirrors. The OTE contributes to the overall wavefront error (and therefore the aberrations in instrument PSFs) in a few ways:
+
```

Comment:
I can add that! (I was trying to avoid listing every optic... any contributions from the pickoffs are also included in the OPD right?)